Sources/Sandbox.Game/Game/Entities/MyEntity.cs

```diff
@@ -477,17 +504,32 @@ public void CreateSync()
         public virtual void UpdateOnceBeforeFrame()
         {
             m_gameLogic.UpdateOnceBeforeFrame();
```

Comment:
Beside the `Update*` methods there's also Init, MarkForClose, Close and UpdatingStopped that need to be handled.